### PR TITLE
Fix code blocks using multiple <code> elements

### LINKED ISSUE #71: Mark code blocks as white-space: pre
At the moment, they aren't, which makes this..

```
for (let i = 0; i < 100; i++) {
  alert(
    'haha pranked' + i
  )
}
```

..display as this:

```
for (let i = 0; i < 100; i++) {
alert(
'haha pranked' + i
)
}
```

### PR DESCRIPTION
**Pull request by heyitsmeuralex**
Also fixes #71

### CODE REVIEW

File: site/styles.css [hunk 1]
```diff
@@ -391,6 +391,9 @@ body, html, #app {
       font-weight: bold;
       color: var(--text-red); }
 
+    .message-inline-code {
+      white-space: pre; }
+
```
Comment (towerofnix) [inline site/styles.css]: So what I'm getting is that we support `\n` in `` `abc` ``-style code blocks, and that ```` ``` ```` code blocks just aren't a thing. I wonder if this should be changed - using two line breaks in a `` `code` `` element `isn't

supported` in Markdown.

In any case, this class name definitely dosen't make sense!

```
blah:

`
asdf
baz
batr
`
```

^ This definitely isn't inline code, is it :)

Comment (heyitsmeuralex) [reply site/styles.css]: >So what I'm getting is that we support \n in `abc`-style code blocks,

Unintentional, but yes. And there's not really much point in fixing it in my opinion.

> ``` code blocks just aren't a thing

They are-- try them.

>  this class name definitely dosen't make sense!

Will fix

Comment (towerofnix) [reply site/styles.css]: >> triple-backslash code blocks just aren't a thing
>
> They are-- try them.

Hm? I don't see any code written to handle ```` ``` ````. Sure, using that string of characters *works*, but I don't think *that's* intentional - rather it just ignores the first two ticks and treats the third as the beginning of the "inline" code-block.

Comment (heyitsmeuralex) [reply site/styles.css]: 100% intentional 😜 

I _thought_ I implemented it at some point but clearly not.

**Comment by heyitsmeuralex** [comment]
I'll probably end up rewriting formatting. It was a really quick job before and it has some pretty big flaws.

It doesn't tokenize or anything which is a bit bad 🌮

**Comment by towerofnix** [comment]
>It doesn't tokenize or anything which is a bit bad :taco:

\>`startToken`

wow is you're code lying :anger:

**Pull request merged**